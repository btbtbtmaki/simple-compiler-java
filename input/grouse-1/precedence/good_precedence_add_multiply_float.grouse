main {

  print 7.2 + 2.4 * 1.0 nl;
  print 4.9 + 6.1 * 2.9 nl;
  print 9.0 + 3.7 * 6.1 nl;
  print 7.5 + 3.1 * 5.2 nl;
  print 3.0 + 9.6 * 6.4 nl;
  print 2.5 + 7.2 * 9.8 nl;
  print 2.4 + 4.2 * 3.6 nl;
  print 8.7 + 4.0 * 4.2 nl;
  print 8.4 + 8.9 * 7.1 nl;
  print 3.0 + 2.9 * 4.4 nl;
  print 5.3 + 3.4 * 8.2 nl;
  print 4.4 + 7.4 * 3.1 nl;

  print 8.4 * 5.6 + 9.9 nl;
  print 6.8 * 2.4 + 2.6 nl;
  print 4.6 * 8.4 + 7.5 nl;
  print 6.4 * 3.3 + 7.9 nl;
  print 6.0 * 6.0 + 5.3 nl;
  print 5.0 * 8.9 + 3.5 nl;
  print 1.3 * 6.5 + 3.3 nl;
  print 9.3 * 9.4 + 1.2 nl;
  print 7.0 * 2.9 + 7.7 nl;
  print 3.0 * 5.5 + 8.1 nl;
  print 1.4 * 9.9 + 5.0 nl;
  print 7.1 * 9.8 + 4.8 nl;

  print 1.9 + 4.3 * 5.2 + 9.4 nl;
  print 8.3 + 3.5 * 7.9 + 7.8 nl;
  print 6.7 + 8.5 * 7.6 + 2.6 nl;
  print 3.5 + 9.8 * 1.7 + 3.1 nl;
  print 2.9 + 3.2 * 7.5 + 5.4 nl;
  print 5.3 + 2.1 * 6.8 + 3.9 nl;
  print 6.4 + 2.8 * 7.1 + 1.8 nl;
  print 3.4 + 3.3 * 5.7 + 6.2 nl;
  print 4.9 + 1.9 * 8.5 + 8.5 nl;
  print 3.2 + 7.4 * 6.7 + 2.5 nl;
  print 7.7 + 5.2 * 3.5 + 9.1 nl;
  print 2.4 + 9.3 * 3.4 + 4.5 nl;

  print 9.8 * 9.5 + 5.1 * 4.3 nl;
  print 8.4 * 7.0 + 8.0 * 4.4 nl;
  print 9.9 * 2.7 + 3.5 * 6.7 nl;
  print 4.7 * 8.1 + 8.9 * 3.6 nl;
  print 6.0 * 4.9 + 2.0 * 4.4 nl;
  print 7.3 * 5.2 + 9.6 * 1.8 nl;
  print 8.1 * 7.6 + 7.3 * 6.4 nl;
  print 8.1 * 5.0 + 3.6 * 9.3 nl;
  print 9.9 * 7.0 + 3.0 * 5.0 nl;
  print 6.1 * 8.5 + 7.4 * 6.4 nl;
  print 6.8 * 7.7 + 5.1 * 8.4 nl;
  print 5.3 * 7.3 + 2.2 * 7.1 nl;

}

