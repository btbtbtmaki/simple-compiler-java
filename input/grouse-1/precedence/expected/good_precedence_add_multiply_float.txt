9.6
22.59
31.57
23.62
64.44
73.06
17.52
25.5
71.59
15.76
33.18
27.34
56.94
18.92
46.14
29.02
41.3
48
11.75
88.62
28
24.6
18.86
74.38
33.66
43.75
73.9
23.26
32.3
23.48
28.08
28.41
29.55
55.28
35
38.52
115.03
94
50.18
70.11
38.2
55.24
108.28
73.98
84.3
99.21
95.2
54.31
